tests/doctests/wfs_MapServerWFSCapabilities.txt

```diff
@@ -101,6 +101,6 @@ Test exceptions
 Lastly, test the getcapabilities method
 
     >>> wfs = WebFeatureService('http://nsidc.org/cgi-bin/atlas_south?', version='1.0')
-    >>> xml = wfs.getcapabilities().read().decode('utf-8')
+    >>> xml = wfs.getcapabilities().read().decode('ISO-8859-1')
     >>> xml.find('<WFS_Capabilities') > 0
```

Comment:
I've added `b` prefix and now this test works fine. Travis shows some fails but it is due to connection problems.